{"name": "test_update_existing_customer", "status": "broken", "statusDetails": {"message": "NameError: name 'generate_random_string' is not defined", "trace": "@pytest.fixture\n    def test_customer_data():\n        return {\n            \"firstname\": \"Test\",\n            \"lastname\": \"User\",\n>           \"email\": generate_random_email(),\n            \"telephone\": generate_random_phone(),\n            \"password\": \"hashed_pass\",  # Предполагем, что пароль уже хэширован\n            \"newsletter\": 0,\n            \"status\": 1,\n        }\n\ntests/test_customer.py:24: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def generate_random_email():\n>       return f\"{generate_random_string()}@example.com\"\nE       NameError: name 'generate_random_string' is not defined\n\ntests/test_customer.py:14: NameError"}, "start": 1746287017594, "stop": 1746287017594, "uuid": "81c811c1-f843-4f44-8147-3c6ea99cbaa1", "historyId": "b2b1532b61fe997b7ba97520860addcb", "testCaseId": "b2b1532b61fe997b7ba97520860addcb", "fullName": "tests.test_customer#test_update_existing_customer", "labels": [{"name": "story", "value": "Обновление клиента"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Клиенты"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_customer"}, {"name": "host", "value": "MacBook-Air-2.local"}, {"name": "thread", "value": "14632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_customer"}]}